tdp/cli/utils.py

```diff
@@ -72,6 +72,14 @@ def dry(func: FC) -> FC:
     )(func)
 
 
+def mock(fun: FC) -> FC:
+    return click.option(
+        "--mock",
```

Comment:
`--mock-deploy` is better, this option mock the only the deployment, not the database for example.

You should add `envvar=TDP_MOCK_DEPLOY` too.

Reply:
You're right. I did it and then force pushed.